src/components/Login/Login.jsx

```diff
@@ -31,9 +31,7 @@ export class Login extends React.Component {
     checkUsernameAllowed(string) {
         // eslint-disable-next-line
         const allowed = /^[A-Za-z0-9]*$/;
-        if (allowed.test(string)){
-            return true;
-        }
+        if (allowed.test(string)) return true;
         return false;
```

Comment:
Can we change this bit to:
`return allowed.test(string)`
?
Or something similar?